wls-gui-wahllokalsystem/src/components/broadcast/TheBroadcastReadConfirmationDialog.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-dialog
+    :model-value="hasBroadcastnachricht"
+    max-width="400px"
+    persistent
+    data-test="dialog-show-broadcast-nachricht"
+  >
+    <v-card>
+      <v-card-title>
+        <v-icon icon="$information"></v-icon>
```

Comment:
```suggestion
        <v-icon icon="$information" size="x-small"></v-icon>
```
ich finde es schöner wenn es kleiner ist weil es sonst so übermächtig wirkt

Reply:
sieht damit besser aus, danke - 8b0f73c56987f6393101545797b1f892b0e01666